docs/usage/self-hosted-configuration.md

```diff
@@ -324,31 +324,92 @@ This private key is used to decrypt config files.
 The corresponding public key can be used to create encrypted values for config files.
 If you want a simple UI to encrypt values you can put the public key in a HTML page similar to <https://renovatebot.com/encrypt>.
```

Comment:
PR title says: `pgp`
In docs we use `gpg`

Not sure which PR title is actually correct here...